Client/Frontend/Reader/ReaderMode.swift

```diff
@@ -135,4 +183,38 @@ class ReaderMode: BrowserHelper {
             }
         }
     }
+
+    private func generateReaderContent(readabilityResult: ReadabilityResult, initialStyle: ReaderModeStyle) -> String? {
+        if let stylePath = NSBundle.mainBundle().pathForResource("Reader", ofType: "css") {
+            if let css = NSString(contentsOfFile: stylePath, encoding: NSUTF8StringEncoding, error: nil) {
+                if let tmplPath = NSBundle.mainBundle().pathForResource("Reader", ofType: "html") {
+                    if let tmpl = NSMutableString(contentsOfFile: tmplPath, encoding: NSUTF8StringEncoding, error: nil) {
```

Comment:
Yuck. I'd prefer doing this instead:
```
let stylePath = ...
let css = ...
if stylePath == nil || css == nil || ... {
    return nil
}
```
just to avoid all this nesting, but your call.